AmericanMedicalAssociation/AMA-style-guide
# AMA-22 (News Item Standard row - Image)

https://palantir.atlassian.net/browse/AMA-22

The 3x2 image was already there from my demo work. I renamed it to group it with the other landscape images in its directory, and adjusted the other file names accordingly to preserve the order.

Steps to test:

1. Atoms > Images > Landscape 3x2 - observe that it exists
2. Verify that the file order in atoms/images seems correct.
